Install and build error with Node 12

When installing the latest NPM package with Node.js v 12.6.0 installed, I get the below error:

```
C:\WINDOWS\system32>npm install -g heap-profile

> heap-profile@0.3.2 install C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile
> node-gyp rebuild


C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile>if not defined npm_config_node_gyp (node "C:\Users\gaura\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Users\gaura\AppData\Roaming\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  sampling-heap-profiler.cc
  win_delay_load_hook.cc
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(26): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(28): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(30): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(32): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(34): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(38): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3367): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(40): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(46): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(48): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(50): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3367): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(52): warning C4996: 'v8
::Object::Set': was declared deprecated [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\build\sampling_he
ap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(3358): note: see declaration of 'v8::Object::Set
  '
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(65): error C2660: 'v8::
Value::Uint32Value': function does not take 0 arguments [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\b
uild\sampling_heap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(2567): note: see declaration of 'v8::Value::Uint
  32Value' (compiling source file ..\bindings\sampling-heap-profiler.cc)
c:\users\gaura\appdata\roaming\npm\node_modules\heap-profile\bindings\sampling-heap-profiler.cc(66): error C2660: 'v8::
Value::IntegerValue': function does not take 0 arguments [C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile\
build\sampling_heap_profiler.vcxproj]
  c:\users\gaura\appdata\local\node-gyp\cache\12.6.0\include\node\v8.h(2565): note: see declaration of 'v8::Value::Inte
  gerValue' (compiling source file ..\bindings\sampling-heap-profiler.cc)
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\gaura\AppData\Roaming\npm\node_modules\node-gyp\lib\build.js:190:23)
gyp ERR! stack     at ChildProcess.emit (events.js:203:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Windows_NT 10.0.18362
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\gaura\\AppData\\Roaming\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\gaura\AppData\Roaming\npm\node_modules\heap-profile
gyp ERR! node -v v12.6.0
gyp ERR! node-gyp -v v5.0.2
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! heap-profile@0.3.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the heap-profile@0.3.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\gaura\AppData\Roaming\npm-cache\_logs\2019-07-07T12_28_22_823Z-debug.log
```

--- a/bindings/sampling-heap-profiler.cc
+++ b/bindings/sampling-heap-profiler.cc
@@ -62,8 +62,8 @@ NAN_METHOD(StartSamplingHeapProfiler) {
     if (!info[1]->IsNumber()) {
       return Nan::ThrowTypeError("Second argument type must be an integer.");
     }
-    uint64_t sample_interval = info[0].As<Integer>()->Uint32Value();
-    int stack_depth = info[1].As<Integer>()->IntegerValue();
+    uint64_t sample_interval = info[0].As<Integer>()->Uint32Value(Nan::GetCurrentContext()).FromJust();
+    int stack_depth = info[1].As<Integer>()->IntegerValue(Nan::GetCurrentContext()).FromJust();
     info.GetIsolate()->GetHeapProfiler()->
       StartSamplingHeapProfiler(sample_interval, stack_depth);
   } else {

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "bindings": "^1.5.0",
-    "nan": "^2.13.2",
+    "nan": "^2.14.0",
     "pify": "^3.0.0"
   },
   "devDependencies": {

